src/DeterministicNoiseSimulator.cpp

```diff
@@ -618,52 +680,4 @@ std::string DeterministicNoiseSimulator::intToString(long target_number, char va
         number = number >> 1u;
     }
     return path;
-}
-
-void DeterministicNoiseSimulator::noiseInsert(const qc::MatrixDD& a, const std::vector<signed char>& usedQubits, const qc::MatrixDD& r) {
-    dd::ComplexValue    aw{dd::CTEntry::val(a.w.r), dd::CTEntry::val(a.w.i)};
-    const unsigned long i = noiseHash(a.p, aw, usedQubits);
-
-    assert(((std::uintptr_t)r.w.r & 1u) == 0 && ((std::uintptr_t)r.w.i & 1u) == 0);
-
-    std::copy(usedQubits.begin(), usedQubits.end(), NoiseTable[i].usedQubits);
-    NoiseTable[i].a    = a.p;
-    NoiseTable[i].aw   = aw;
-    NoiseTable[i].r    = r.p;
-    NoiseTable[i].rw.r = r.w.r->value;
-    NoiseTable[i].rw.i = r.w.i->value;
-}
-
-qc::MatrixDD DeterministicNoiseSimulator::noiseLookup(const qc::MatrixDD& a, const std::vector<signed char>& usedQubits) {
-    // Lookup a computation in the compute table
-    // return NULL if not a match else returns result of prior computation
-    qc::MatrixDD r{nullptr, {nullptr, nullptr}};
-
-    dd::ComplexValue aw{dd::CTEntry::val(a.w.r), dd::CTEntry::val(a.w.i)};
-
-    const unsigned long i = noiseHash(a.p, aw, usedQubits);
-
-    if (NoiseTable[i].a != a.p || NoiseTable[i].aw != aw) return r;
-    if (!std::equal(usedQubits.begin(), usedQubits.end(), std::begin(NoiseTable[i].usedQubits))) {
-        return r;
-    }
-
-    r.p = NoiseTable[i].r;
-
-    if (std::abs(NoiseTable[i].rw.r) < dd::ComplexTable<>::tolerance() && std::fabs(NoiseTable[i].rw.i) < dd::ComplexTable<>::tolerance()) {
-        return qc::MatrixDD::zero;
-    } else {
-        r.w = dd->cn.getCached(NoiseTable[i].rw.r, NoiseTable[i].rw.i);
-    }
-    return r;
-}
-
-unsigned long DeterministicNoiseSimulator::noiseHash(dd::mNode* a, const dd::ComplexValue& aw, const std::vector<signed char>& usedQubits) {
-    unsigned long i = 0;
-
-    for (auto qubit: usedQubits) {
-        i = (i << 3U) + i * qubit + qubit;
-    }
-    i += (reinterpret_cast<std::uintptr_t>(a)) + static_cast<unsigned long>(aw.r * 1000000 + aw.i * 2000000);
-    return i & NoiseMASK;
-}
+}
```

Comment:
there is a newline missing at the end of this file

Reply:
Added the newline